app/upgrade_cudos.go

```diff
@@ -1988,16 +1988,13 @@ func DoGenesisAccountMovements(genesisData *GenesisData, cudosCfg *CudosMergeCon
 			return nil
 		}
 
-		var totalAmountToMove sdk.Int
 		fromAccTokensAmount := fromAcc.balance.AmountOfNoDenomValidation(genesisData.bondDenom)
 
 		// Move entire balance if balance to move is 0 or greater than available balance
 		if accountMovement.Amount.IsZero() || fromAccTokensAmount.LT(accountMovement.Amount) {
```

Comment:
I think we can/shound use here `LTE(...)` instead of `LT(...)`: 
```suggestion
if accountMovement.Amount.IsZero() || fromAccTokensAmount.LTE(accountMovement.Amount) {
```